mta/tests/test_projects.py

```diff
@@ -98,3 +98,15 @@ def test_search_project(create_minimal_project):
     project_collection.search_project(project.name)
     view = project_collection.create_view(AllProjectView)
     assert project.name in [row.name.text for row in view.table]
+
+
+def test_send_feedback(create_minimal_project):
```

Comment:
@ganeshhubale : We need to write other tests too for reports page .so we can create a new file test_report.py